config/router.config.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+
+const controller = require("../controllers/spotify.controller.js");
```

Comment:
```suggestion
const spotifyController = require("../controllers/spotify.controller.js");
```

Piensa que ahora mismo solo tenemos un controlador, pero esto es escalable! asi que si nos acostumbramos desde ya a poner nombres a las variables que sea descriptivos vuestro valor y conocimiento crecera x1000, 0 bromas! :D